src/Map/.gitignore

```diff
@@ -0,0 +1,4 @@
+vendor
+composer.lock
+.php_cs.cache
```

Comment:
```suggestion
```

Reply:
Note that this file is also mentioned in other `.gitignore` under `src/*`

Reply:
Follow-up: #1969